tests/test_io.py

```diff
@@ -318,11 +318,11 @@ def test_save_folder_grainstats(tmp_path: Path) -> None:
     assert Path(out_path / "processed" / "folder_grainstats.csv").exists()
 
 
-def test_load_scan_spm(load_scan: LoadScans) -> None:
+def test_load_scan_spm(load_scan_spm: LoadScans) -> None:
```

Comment:
If we're switching to `minicricle_small` for the vast majority of tests do you think we should switch it for the `load_scan_spm` here too?

Or is it not practical because `minicircle_small` is of type `.topostats` and making the equivalent `.spm` is not possible/a lot of hassle?

Reply:
I tried my best to construct a `.spm` from `minicircle_small`, but Gwyddion does now allow saving to `.spm`, so it's effectively a read-only file. 😢